snippets/stableSort.md

```diff
@@ -0,0 +1,29 @@
+### stableSort
```

Comment:
We need a different name for this than stable sort since it's an indexed relative sort, and that's to long a name. MAybe call it indexedSort or preservedSort?

Reply:
Don't forget to rename the snippet file as well